src/main/java/org/victorrobotics/frc/dtlib/actuator/motor/DTMotorFaults.java

```diff
@@ -1,7 +1,7 @@
 package org.victorrobotics.frc.dtlib.actuator.motor;
 
 public interface DTMotorFaults<FAULT_TYPE> {
-    FAULT_TYPE internal();
+    FAULT_TYPE getFaultsImpl();
```

Comment:
Not sure this is needed since the motor fault objects both provide the same methods.

Reply:
Maybe, but in some cases it might be nice to get something more specific. I suppose you probably could do `DTMotor.getMotorImpl().[faults-method]` though.

Reply:
Sorry, I wasn't clear. I meant that the parameter wasn't needed because both concrete fault classes have the same methods. Having the internal fault object doesn't buy you anything.